apps/web/src/services/asset-inventory/collector/modules/CollectorContents.vue

```diff
@@ -26,7 +26,7 @@
         </p-toolbox>
         <div class="collector-list-wrapper">
             <div
-                v-if="cloudCollectorPageState.filteredList.length > 0"
+                v-if="cloudCollectorPageState.filteredList?.length > 0"
```

Comment:
question: Does optional chaining(`?`) works in Vue2.7? 😯

Reply:
I'm sorry! I missed it!! Let me correct myself!